### PR TITLE
add invFlags, eveUnits and typeIDs tables/files

### PR DESCRIPTION
**Pull request by deadlybulb**
Added the invFlags table with this update.  Also took a crack at creating a description of typeIDs.yaml.  Many unknowns on this one:

* I have no idea what the orderID column is for on invFlags
* Please double check the units (kilograms, cubic meters, etc.) on yaml_typeIDs.md
* I'm not sure what "portionSize" is on typeIDs, nor what the exact units should be (maybe unitless)
* Not quite sure if I got "basePrice" right in typeIDs
* I have no idea what "sofFactionName" and "sofMaterialSetID" are in typeIDs
* A made a guess on the key for the "traits" map in typeIDs.  I'm not sure my description is correct.
* I don't know what unitID is in the bonus object mapped from traits in typeIDs.

I tried describing typeIDs.yaml using a schema format.  Unfortunately, there's no agreed upon schema language for YAML so I made this up a bit here.

### CODE REVIEW

**Comment by Regner** [comment]
I just merged a PR from @CaptainThunk that included blueprints.

**Comment by deadlybulb** [comment]
Doh!  Ok, pulled in latest upstream and merged.  Also changed the name of this PR to more accurately reflect what would be merged.  Is blueprint.flagID really just a constant?  I thought it corresponded to invFlags.

**Comment by CaptainThunk** [comment]
unitID references dbo.eveUnits

portionSize is a reference to how many of the item comes in a batch - eg, for mining it's the unmodified amount that appears in your hangar, for production it's the number of units you produce per run (eg, 100 for ammo) - needs confirmation and may be legacy stuff as these things have had several revamps over the years

**Comment by deadlybulb** [comment]
Updated unitID and portionSize per CaptainThunk.  Also added the eveUnits table docs.

File: mkdocs.yml [hunk 1]
```diff
@@ -40,7 +40,6 @@ pages:
 - ['xmlapi/char_contractitems.md', 'XML API', 'Char - ContractItems']
 - ['xmlapi/char_contracts.md', 'XML API', 'Char - Contracts']
 - ['xmlapi/char_marketorders.md', 'XML API', 'Char - MarketOrders']
-- ['xmlapi/char_upcomingcalendarevents.md', 'XML API', 'Char - UpcomingCalendarEvents']
```
Comment (deadlybulb) [inline mkdocs.yml]: mkdocs was unhappy unless I removed this line (file is not present yet).

**Comment by JimiC** [comment]
"sofFactionName" and "sofMaterialSetID" are references to skins (skins.yaml).

**Comment by deadlybulb** [comment]
Awesome.  I'll add these in my next PR.

**Pull request merged**